packages/command/src/sfdxCommand.ts

```diff
@@ -265,6 +278,22 @@ export abstract class SfdxCommand extends Command {
     return true;
   }
 
+  protected async hooksFromLifecycleEvent(lifecycleEventNames: string[]) {
```

Comment:
This could probably be private since commands are really just going to set the property.